app/main/views/audits.py

```diff
@@ -39,3 +67,32 @@ def list_audits():
             request.args
         )
     )
+
+
+@main.route('/audit-events/<string:audit_id>/acknowledge', methods=['POST'])
```

Comment:
Gives a 404 not a 400. Was thinking that the bad request response was preferable. Thoughts?